src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -0,0 +1,80 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.kd-replicasetdetail-sidebar {
+  background-color: #FAFAFA;
+  min-width: 310px;
+}
+
+.kd-replicasetdetail-table td,
+.kd-replicasetdetail-table th {
+  text-align: left;
+  color: #777777;
+  font-size: 13px;
+  border-bottom: 1px solid #CCCCCC;
+}
+
+.kd-replicasetdetail-table-dark {
+  color: #000000 !important;
+}
+
+.kd-replicasetdetail-table .material-icons {
+  vertical-align: middle;
+  padding: 0 5px 0 5px;
+  font-size: 13px;
+}
+
+.kd-align-mid {
+  vertical-align: middle;
+}
+
+.kd-font-lg {
+  font-size: 2em;
+}
+
+.kd-header-option-panel {
+  color: dimgray;
+  border-bottom: 1px solid #DDDDDD;
+  min-height: 48px;
+  max-height: 48px;
+}
+
+.kd-primary {
+  color: #326de6;
+}
+
+.kd-font-1 {
+  color: #AAAAAA;
```

Comment:
I see that this CSS contains like 5 colors. Can you reduce their number? If not, extract to constants to give meaning?